<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateOrder</class>
 <widget class="QDialog" name="CreateOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Utwórz zlecenie</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>2</y>
     <width>398</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="empty_data_label">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="mark_label">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Marka</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mark_input"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="model_label">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Model</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="model_input"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="year_label">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Rok Produkcji</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="year_input"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="volume_label">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Pojemność Silnika</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="volume_engine"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="type_label">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Typ Silnika</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="type_input">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>DIESEL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>BENZYNA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LPG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ELEKTRYCZNY</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HYBRYDA</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="vin_label">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Vin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="vin_input"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="rej_label">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nr rejestracyjny</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="rej_input"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="desc_label">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Opis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="desc_input">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>UTWÓRZ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
